Docs: Error tracking polish (#11002)
PostHog/posthog.com: contents/docs/error-tracking/_snippets/manual-error-tracking.mdx

If you're using a different SDK or the API, you can manually capture errors by using the capture API or calling the `capture` method to capture an `$exception` event with the following properties:

| Property | Description |
|----------|-------------|
| `$exception_list` | A list of exception objects with detailed information about each error |

> | `$exception_fingerprint` | A unique hash identifying the exception pattern |

Reviewer: ```suggestion
| `$exception_fingerprint` | (Optional) The identifier used to group issues. If not set, a unique hash based on the exception pattern will be generated during ingestion |
```

> | `$exception_proposed_fingerprint` | Suggested fingerprint for grouping similar exceptions |

Reviewer: This is added by cymbal
```suggestion
```

> | `$exception_issue_id` | Unique identifier for the exception instance (`c5d4387a-1c32-4132-9511-264268a8099f`) |

Reviewer: This is chosen in cymbal
```suggestion
```

> | `$exception_level` | Severity level of the exception (e.g., `error`) |
> | `$exception_types` | List of error types encountered (e.g., `["RangeError"]`) |
> | `$exception_values` | List of error messages (e.g., `["Maximum call stack size exceeded"]`) |
> | `$exception_sources` | List of source files where exceptions occurred |
> | `$exception_functions` | List of functions involved in the exception |

Reviewer: All added during ingestion
```suggestion
```

> | `$exception_personURL` | URL to view the affected user's profile |

Reviewer: I think we only added this in Sentry
```suggestion
```

> | `$exception_capture_enabled_server_side` | Flag indicating if exception capturing is enabled |

Reviewer: Don't know if we care about this one either for ingestion
```suggestion
```
